schemas/resources/schemas/provider/v1.0.0/ingest.md

```diff
@@ -84,19 +84,19 @@ The contents of the metadata is passed in the body of the request.
     }
 ```
 
-### <a name="delete-grid"></a> DELETE - Delete a Provider
+### <a name="delete-provider"></a> DELETE - Delete a Provider
 
 Not currently allowed. Contact CMR support.
 
 <!--
-Grid metadata can be deleted by sending an HTTP DELETE to the URL
+Provider metadata can be deleted by sending an HTTP DELETE to the URL
 `%CMR-ENDPOINT%/providers`.
 
 ```
     curl -XDELETE \
         -H "Cmr-Pretty: true" \
         -H "Authorization: Bearer XXXX" \
-        %CMR-ENDPOINT%/providers/PROV1/grids/sampleNative23Id"
+        %CMR-ENDPOINT%/providers/PROV1/provider/prov-id-here"
```

Comment:
nit: maybe you could wrap that in <> so <prov-id> but, if this is more clear to you I defer my opinion

Reply:
< and > cause problems with some markdown converters because it looks like a tag